src/main.scss

```diff
@@ -6,12 +6,14 @@
 @include font-face("SourceSansPro", "assets/fonts/sourcesanspro-regular-webfont", normal, normal);
 @include font-face("SourceSansPro", "assets/fonts/sourcesanspro-semibold-webfont", 600, normal);
 
+@import "src/styles/font-styles";
+@import "src/styles/forms";
+@import "src/styles/buttons";
```

Comment:
theses files should be attached to their component. The only I can see that is global related is font-styles.

Reply:
forms and buttons are meant to be used throughout various components. I really have to call them in each component when they are needed ?

Reply:
@KamiAmiga If it's embed in the component , for example button component use buttons.scss then every time you use button.vue component its style will follow.